23:26:23 ConfigMap 08db1feb.percona.com 847:	percona-xtradb-cluster-operator-6db54dc756-7mx7w_5346714d-0fcd-4086-907e-1527c0532974 became leader
23:26:23 Lease 08db1feb.percona.com 848:	percona-xtradb-cluster-operator-6db54dc756-7mx7w_5346714d-0fcd-4086-907e-1527c0532974 became leader
23:26:49 ConfigMap 08db1feb.percona.com 913:	percona-xtradb-cluster-operator-64c4fd5974-4bz5x_ba03052e-5c98-4571-90b6-80878a574c06 became leader
23:26:49 Lease 08db1feb.percona.com 914:	percona-xtradb-cluster-operator-64c4fd5974-4bz5x_ba03052e-5c98-4571-90b6-80878a574c06 became leader
23:32:24 PersistentVolumeClaim datadir-test-cluster-pxc-0 1760:	waiting for first consumer to be created before binding
23:32:24 PersistentVolumeClaim datadir-test-cluster-pxc-0 1768:	waiting for a volume to be created, either by external provisioner "rancher.io/local-path" or manually created by system administrator
23:32:24 PersistentVolumeClaim datadir-test-cluster-pxc-0 1768:	External provisioner is provisioning volume for claim "acto-namespace/datadir-test-cluster-pxc-0"
23:32:28 PersistentVolumeClaim datadir-test-cluster-pxc-0 1768:	Successfully provisioned volume pvc-7123ea87-1004-4e89-8fae-8ee647f53db9
23:33:29 PersistentVolumeClaim datadir-test-cluster-pxc-1 2082:	waiting for first consumer to be created before binding
23:33:29 PersistentVolumeClaim datadir-test-cluster-pxc-1 2088:	waiting for a volume to be created, either by external provisioner "rancher.io/local-path" or manually created by system administrator
23:33:29 PersistentVolumeClaim datadir-test-cluster-pxc-1 2088:	External provisioner is provisioning volume for claim "acto-namespace/datadir-test-cluster-pxc-1"
23:33:34 PersistentVolumeClaim datadir-test-cluster-pxc-1 2088:	Successfully provisioned volume pvc-45905ae9-d69c-4e03-8853-0597505ffba5
23:34:35 PersistentVolumeClaim datadir-test-cluster-pxc-2 2383:	waiting for first consumer to be created before binding
23:34:35 PersistentVolumeClaim datadir-test-cluster-pxc-2 2390:	waiting for a volume to be created, either by external provisioner "rancher.io/local-path" or manually created by system administrator
23:34:35 PersistentVolumeClaim datadir-test-cluster-pxc-2 2390:	External provisioner is provisioning volume for claim "acto-namespace/datadir-test-cluster-pxc-2"
23:34:40 PersistentVolumeClaim datadir-test-cluster-pxc-2 2390:	Successfully provisioned volume pvc-0b6d2af0-8b41-4c71-9fd7-52c4e8bacc20
23:26:23 Pod percona-xtradb-cluster-operator-64c4fd5974-4bz5x 827:	Successfully assigned acto-namespace/percona-xtradb-cluster-operator-64c4fd5974-4bz5x to acto-cluster-0-worker2
23:26:24 Pod percona-xtradb-cluster-operator-64c4fd5974-4bz5x 830:	Pulling image "percona/percona-xtradb-cluster-operator:1.11.0"
23:26:25 Pod percona-xtradb-cluster-operator-64c4fd5974-4bz5x 830:	Successfully pulled image "percona/percona-xtradb-cluster-operator:1.11.0" in 584.970081ms
23:26:25 Pod percona-xtradb-cluster-operator-64c4fd5974-4bz5x 830:	Created container percona-xtradb-cluster-operator
23:26:26 Pod percona-xtradb-cluster-operator-64c4fd5974-4bz5x 830:	Started container percona-xtradb-cluster-operator
23:26:23 ReplicaSet percona-xtradb-cluster-operator-64c4fd5974 825:	Created pod: percona-xtradb-cluster-operator-64c4fd5974-4bz5x
23:26:13 Pod percona-xtradb-cluster-operator-6db54dc756-7mx7w 779:	Successfully assigned acto-namespace/percona-xtradb-cluster-operator-6db54dc756-7mx7w to acto-cluster-0-worker
23:26:14 Pod percona-xtradb-cluster-operator-6db54dc756-7mx7w 782:	Pulling image "percona/percona-xtradb-cluster-operator:1.11.0"
23:26:14 Pod percona-xtradb-cluster-operator-6db54dc756-7mx7w 782:	Successfully pulled image "percona/percona-xtradb-cluster-operator:1.11.0" in 537.818522ms
23:26:14 Pod percona-xtradb-cluster-operator-6db54dc756-7mx7w 782:	Created container percona-xtradb-cluster-operator
23:26:15 Pod percona-xtradb-cluster-operator-6db54dc756-7mx7w 782:	Started container percona-xtradb-cluster-operator
23:26:23 Pod percona-xtradb-cluster-operator-6db54dc756-7mx7w 782:	Stopping container percona-xtradb-cluster-operator
23:26:13 ReplicaSet percona-xtradb-cluster-operator-6db54dc756 776:	Created pod: percona-xtradb-cluster-operator-6db54dc756-7mx7w
23:26:23 ReplicaSet percona-xtradb-cluster-operator-6db54dc756 833:	Deleted pod: percona-xtradb-cluster-operator-6db54dc756-7mx7w
23:26:13 Deployment percona-xtradb-cluster-operator 775:	Scaled up replica set percona-xtradb-cluster-operator-6db54dc756 to 1
23:26:23 Deployment percona-xtradb-cluster-operator 824:	Scaled up replica set percona-xtradb-cluster-operator-64c4fd5974 to 1
23:26:23 Deployment percona-xtradb-cluster-operator 824:	Scaled down replica set percona-xtradb-cluster-operator-6db54dc756 to 0
23:32:24 Pod test-cluster-haproxy-0 1793:	Successfully assigned acto-namespace/test-cluster-haproxy-0 to acto-cluster-0-worker3
23:32:25 Pod test-cluster-haproxy-0 1798:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
23:32:25 Pod test-cluster-haproxy-0 1798:	Created container haproxy
23:32:25 Pod test-cluster-haproxy-0 1798:	Started container haproxy
23:32:25 Pod test-cluster-haproxy-0 1798:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
23:32:25 Pod test-cluster-haproxy-0 1798:	Created container pxc-monit
23:32:25 Pod test-cluster-haproxy-0 1798:	Started container pxc-monit
23:32:44 Pod test-cluster-haproxy-0 1798:	Readiness probe failed: ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 2

23:33:54 Pod test-cluster-haproxy-0 1798:	Liveness probe failed: ERROR 2013 (HY000): Lost connection to MySQL server at 'reading initial communication packet', system error: 2

23:34:00 Pod test-cluster-haproxy-1 2232:	Successfully assigned acto-namespace/test-cluster-haproxy-1 to acto-cluster-0-worker2
23:34:00 Pod test-cluster-haproxy-1 2234:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
23:34:00 Pod test-cluster-haproxy-1 2234:	Created container haproxy
23:34:00 Pod test-cluster-haproxy-1 2234:	Started container haproxy
23:34:00 Pod test-cluster-haproxy-1 2234:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
23:34:00 Pod test-cluster-haproxy-1 2234:	Created container pxc-monit
23:34:01 Pod test-cluster-haproxy-1 2234:	Started container pxc-monit
23:34:15 Pod test-cluster-haproxy-2 2299:	Successfully assigned acto-namespace/test-cluster-haproxy-2 to acto-cluster-0-worker
23:34:15 Pod test-cluster-haproxy-2 2301:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
23:34:15 Pod test-cluster-haproxy-2 2301:	Created container haproxy
23:34:16 Pod test-cluster-haproxy-2 2301:	Started container haproxy
23:34:16 Pod test-cluster-haproxy-2 2301:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-haproxy" already present on machine
23:34:16 Pod test-cluster-haproxy-2 2301:	Created container pxc-monit
23:34:16 Pod test-cluster-haproxy-2 2301:	Started container pxc-monit
23:50:21 Pod test-cluster-haproxy-2 2301:	Stopping container haproxy
23:50:21 Pod test-cluster-haproxy-2 2301:	Stopping container pxc-monit
23:50:22 Pod test-cluster-haproxy-2 5121:	Successfully assigned acto-namespace/test-cluster-haproxy-2 to acto-cluster-0-worker
23:50:23 Pod test-cluster-haproxy-2 5122:	Failed to apply default image tag "ACTOKEY": couldn't parse image reference "ACTOKEY": invalid reference format: repository name must be lowercase
23:50:23 Pod test-cluster-haproxy-2 5122:	Error: InvalidImageName
23:50:23 Pod test-cluster-haproxy-2 5122:	Failed to apply default image tag "ACTOKEY": couldn't parse image reference "ACTOKEY": invalid reference format: repository name must be lowercase
23:50:23 Pod test-cluster-haproxy-2 5122:	Error: InvalidImageName
23:32:24 PodDisruptionBudget test-cluster-haproxy 1789:	No matching pods found
23:32:24 StatefulSet test-cluster-haproxy 1781:	create Pod test-cluster-haproxy-0 in StatefulSet test-cluster-haproxy successful
23:34:00 StatefulSet test-cluster-haproxy 1820:	create Pod test-cluster-haproxy-1 in StatefulSet test-cluster-haproxy successful
23:34:15 StatefulSet test-cluster-haproxy 2235:	create Pod test-cluster-haproxy-2 in StatefulSet test-cluster-haproxy successful
23:50:21 StatefulSet test-cluster-haproxy 5094:	delete Pod test-cluster-haproxy-2 in StatefulSet test-cluster-haproxy successful
23:32:29 Pod test-cluster-pxc-0 1762:	Successfully assigned acto-namespace/test-cluster-pxc-0 to acto-cluster-0-worker
23:32:30 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster-operator:1.11.0" already present on machine
23:32:30 Pod test-cluster-pxc-0 1861:	Created container pxc-init
23:32:31 Pod test-cluster-pxc-0 1861:	Started container pxc-init
23:32:32 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
23:32:32 Pod test-cluster-pxc-0 1861:	Created container logs
23:32:32 Pod test-cluster-pxc-0 1861:	Started container logs
23:32:32 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
23:32:32 Pod test-cluster-pxc-0 1861:	Created container logrotate
23:32:32 Pod test-cluster-pxc-0 1861:	Started container logrotate
23:32:32 Pod test-cluster-pxc-0 1861:	Container image "percona/percona-xtradb-cluster:8.0.27-18.1" already present on machine
23:32:32 Pod test-cluster-pxc-0 1861:	Created container pxc
23:32:33 Pod test-cluster-pxc-0 1861:	Started container pxc
23:32:59 Pod test-cluster-pxc-0 1861:	Readiness probe failed: ERROR 2003 (HY000): Can't connect to MySQL server on '10.244.3.4:33062' (111)
+ [[ '' == \P\r\i\m\a\r\y ]]
+ exit 1

23:33:34 Pod test-cluster-pxc-1 2085:	Successfully assigned acto-namespace/test-cluster-pxc-1 to acto-cluster-0-worker2
23:33:36 Pod test-cluster-pxc-1 2128:	Container image "percona/percona-xtradb-cluster-operator:1.11.0" already present on machine
23:33:36 Pod test-cluster-pxc-1 2128:	Created container pxc-init
23:33:36 Pod test-cluster-pxc-1 2128:	Started container pxc-init
23:33:36 Pod test-cluster-pxc-1 2128:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
23:33:36 Pod test-cluster-pxc-1 2128:	Created container logs
23:33:36 Pod test-cluster-pxc-1 2128:	Started container logs
23:33:36 Pod test-cluster-pxc-1 2128:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
23:33:36 Pod test-cluster-pxc-1 2128:	Created container logrotate
23:33:36 Pod test-cluster-pxc-1 2128:	Started container logrotate
23:33:36 Pod test-cluster-pxc-1 2128:	Container image "percona/percona-xtradb-cluster:8.0.27-18.1" already present on machine
23:33:36 Pod test-cluster-pxc-1 2128:	Created container pxc
23:33:37 Pod test-cluster-pxc-1 2128:	Started container pxc
23:34:05 Pod test-cluster-pxc-1 2128:	Readiness probe failed: ERROR 2003 (HY000): Can't connect to MySQL server on '10.244.1.4:33062' (111)
+ [[ '' == \P\r\i\m\a\r\y ]]
+ exit 1

23:34:41 Pod test-cluster-pxc-2 2386:	Successfully assigned acto-namespace/test-cluster-pxc-2 to acto-cluster-0-worker3
23:34:42 Pod test-cluster-pxc-2 2436:	Container image "percona/percona-xtradb-cluster-operator:1.11.0" already present on machine
23:34:42 Pod test-cluster-pxc-2 2436:	Created container pxc-init
23:34:42 Pod test-cluster-pxc-2 2436:	Started container pxc-init
23:34:42 Pod test-cluster-pxc-2 2436:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
23:34:42 Pod test-cluster-pxc-2 2436:	Created container logs
23:34:42 Pod test-cluster-pxc-2 2436:	Started container logs
23:34:42 Pod test-cluster-pxc-2 2436:	Container image "percona/percona-xtradb-cluster-operator:1.11.0-logcollector" already present on machine
23:34:42 Pod test-cluster-pxc-2 2436:	Created container logrotate
23:34:43 Pod test-cluster-pxc-2 2436:	Started container logrotate
23:34:43 Pod test-cluster-pxc-2 2436:	Container image "percona/percona-xtradb-cluster:8.0.27-18.1" already present on machine
23:34:43 Pod test-cluster-pxc-2 2436:	Created container pxc
23:34:43 Pod test-cluster-pxc-2 2436:	Started container pxc
23:35:11 Pod test-cluster-pxc-2 2436:	Readiness probe failed: ERROR 2003 (HY000): Can't connect to MySQL server on '10.244.2.4:33062' (111)
+ [[ '' == \P\r\i\m\a\r\y ]]
+ exit 1

23:32:24 StatefulSet test-cluster-pxc 1758:	create Claim datadir-test-cluster-pxc-0 Pod test-cluster-pxc-0 in StatefulSet test-cluster-pxc success
23:32:24 StatefulSet test-cluster-pxc 1758:	create Pod test-cluster-pxc-0 in StatefulSet test-cluster-pxc successful
23:33:29 StatefulSet test-cluster-pxc 2063:	create Claim datadir-test-cluster-pxc-1 Pod test-cluster-pxc-1 in StatefulSet test-cluster-pxc success
23:33:29 StatefulSet test-cluster-pxc 2063:	create Pod test-cluster-pxc-1 in StatefulSet test-cluster-pxc successful
23:34:35 StatefulSet test-cluster-pxc 2376:	create Claim datadir-test-cluster-pxc-2 Pod test-cluster-pxc-2 in StatefulSet test-cluster-pxc success
23:34:35 StatefulSet test-cluster-pxc 2376:	create Pod test-cluster-pxc-2 in StatefulSet test-cluster-pxc successful
